my_list.rb

```diff
@@ -0,0 +1,13 @@
+require './my_enumerable'
+
```

Comment:
- [ ] I would suggest using `require_relative` to import `my_enumerable` since it is a ruby convention. Require should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory.

```rb
 require_relative 'my_enumerable'
```